README.md

```diff
@@ -33,6 +33,7 @@ To run the app you need to create a config.json file in the src/security folder
   "client_id": "",
   "client_secret": "",
```

Comment:
To be removed, the secret is only required in the backend.
In fact, when it's supplied, the client will get it and compromise our OAuth2